www/common/sframe-common-mailbox.js

```diff
@@ -81,8 +81,18 @@ define([
                     Common.openURL(Hash.hashToHref('', 'calendar'));
                 });
             } else if (userData && typeof(userData) === "object" && userData.profile) {
-                avatar = h('span.cp-avatar');
+                avatar = h('span.cp-avatar',{
+                    tabindex: 0,
+                    title: Messages.user_profile,
+                    'aria-label': Messages.user_profile,
+                    role: 'button'
+                });
                 Common.displayAvatar($(avatar), userData.avatar, userData.displayName || userData.name);
+                $(avatar).keydown(function (e) {
+                    if (e.which === 13 || e.which === 32) {
+                        $(avatar).click();
+                    }
+                });
```

Comment:
You can use `Util.onClickEnter($(avatar), handler, { space: true });` to avoid triggering a click on Enter or Space